know/rdf/parser/wikidata_parser.py

```diff
@@ -0,0 +1,23 @@
+
+from .abstract_parser import AbstractParser
+
+class WikidataParser(AbstractParser):
+    """parses wikidata sources"""
+
+    def __init__(self, entity_id):
+        self.entity_id = entity_id
+
+    def parse(self) -> dict:
+        """call this method to start parsing"""
+        entity_type = self.get_entity_type()
+        #TODO call correct parsing function based on entity type
+        return dict()
+    def get_entity_type(self) -> str:
+        """returns appropriate entity type as string"""
+        return ""
```

Comment:
We're going to want to make this an Enum, e.g.

```python
from enum import Enum
class EntityType(Enum):
    PERSON = 1
    BOOK = 2
    PAINTING = 3
```

then you can reference it like

`return EntityType.PERSON`

I would default to have this return `None` for now. Also don't forget to make sure to change the return type